modules/microshift-nw-enforcing-hsts-per-domain.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-configuring-routes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-nw-enforcing-hsts-per-domain_{context}"]
+= Enforcing HTTP Strict Transport Security per-domain
+
+You can configure a route with a compliant HSTS policy annotation. To handle upgraded clusters with non-compliant HSTS routes, you can update the manifests at the source and apply the updates.
+
+You cannot use `oc expose route` or `oc create route` commands to add a route in a domain that enforces HSTS because the API for these commands does not accept annotations.
+
+[IMPORTANT]
+====
+HSTS cannot be applied to insecure, or non-TLS routes, even if HSTS is requested for all routes globally.
```

Comment:
The global HSTS happens at the ingress resource from OpenShift, right? If I am not mistaken the second part of the sentence is not possible in MicroShift.